packages/flutter_tools/test/commands.shard/hermetic/build_macos_test.dart

```diff
@@ -191,6 +191,33 @@ void main() {
     Platform: () => macosPlatform,
     FeatureFlags: () => TestFeatureFlags(isMacOSEnabled: true),
   });
+  testUsingContext('macOS build supports build name and number', () async {
+    final BuildCommand command = BuildCommand();
+    createMinimalMockProjectFiles();
+
+    await createTestCommandRunner(command).run(
+      const <String>[
+        'build',
+        'macos',
+        '--debug',
+        '--build-name=1.2.3',
+        '--build-number=42',
+      ],
+    );
+    final String contents = fileSystem.file(
+        './macos/Flutter/ephemeral/Flutter-Generated.xcconfig')
```

Comment:
nit: indent 2 spaces and put `.file` on the same line as the path

Reply:
👍️